eslint-custom-rules/no-ts-expect-error.test.js

```diff
@@ -0,0 +1,42 @@
+const { RuleTester } = require('eslint')
+const rule = require('./no-ts-expect-error')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+})
+
+ruleTester.run('no-ts-expect-error', rule, {
+  valid: [
+    {
+      code: `// @ts-expect-error: something else\nconst a = {};`,
```

Comment:
pour info, il serait aussi possible d'écrire avec des vrais saut de lignes

je trouve ça plus lisible

```suggestion
      code: `
        // @ts-expect-error: something else
        const a = {};
      `,
```